Automattic/woocommerce-payments
# Overview page improvements

Fixes #1973.

#### Changes proposed in this Pull Request

- Hide the pending deposits count when it's zero or not set.
- Use `isNarrow` for the overview page with in order to avoid additional CSS.
- Remove the deposits overview from the standard Deposits page.

#### Testing instructions

Use the fixture data from the original PR: https://github.com/Automattic/woocommerce-payments/pull/1892#discussion_r640652182

1. Play with the `deposits_count` value in the JSON:
    - Remove it
    - Set it to 0
    - Set it to more than 0
2. Check the Deposits page and make sure that it does not contain the overview anymore.
3. On the overview page, make sure that the test mode notice includes more than just a link.

-------------------
